README.md

```diff
@@ -591,15 +591,15 @@ if ($cookieBar) {
 }
 ```
 
-### Set user agent
+### Set User-Agent
```

Comment:
Please revert this title change.

Reply:
Thanks for noticing. I done for required changes.